src/ripple/app/rdb/backend/impl/PostgresDatabase.cpp

```diff
@@ -189,8 +187,6 @@ loadLedgerInfos(
         auto minAndMax =
             std::get_if<std::pair<uint32_t, uint32_t>>(&whichLedger))
     {
-        expNumResults = minAndMax->second - minAndMax->first;
```

Comment:
Is it useful to assert the validity of the output of the Postgres database?
`expNumResults` could be set to an initial absurd value (say `-1`), later we can do:
```
if(expNumResults != -1)
    assert(expNumResults == res.ntuples());
```

Similar to the validation at https://github.com/XRPLF/rippled/blob/3c321f54cc03575f8f85fc1fa73737134f0ada09/src/ripple/app/rdb/backend/impl/PostgresDatabase.cpp#L223

Is this useful?